VMT/KCSA/KCSA-CVE-2019-5736.md

```diff
@@ -0,0 +1,49 @@
+announcement-date: 2019-02-15
+
+id: KCSA-CVE-2019-5736
+
+title: CVE-2019-5736 'runc container breakout' assessment
+
+description: Impact of CVE-2019-5736 on Kata Containers
+
+potentially-affected-components:
+
+  - components: kata-agent
+    version: all
+
+vulnerabilities:
+
+  - cve-id: CVE-2019-5736
+
+reporters:
+
+  - name: Graham Whaley
+    affiliation: VMT member
+    reported:
+      - Original report seen on dev@opencontainers.org
+      - oss-sec posting: https://seclists.org/oss-sec/2019/q1/119
+      - MITRE: https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-5736
+
+issues:
+
+  links:
+    - https://github.com/kata-containers/$COMPONENT/issues/$ISSUE_NR
```

Comment:
We should fill out the component and issue here - probably just pointing to #85 in this repo is enough for this KSCA.

Reply:
oops! fix, thanks!